### ISSUE TITLE
Angular ChoiceInput a11y

### ISSUE THREAD

**Issue by roblevintennis** [AgnosticUI/agnosticui#28]
Looks like there's an a11y violation in the angular checkboxes/radio implementation:

![image](https://user-images.githubusercontent.com/142403/135871198-b274f8bb-afbd-4828-9655-90075160f4ce.png)

**Comment by roblevintennis** [comment]
Similar one for React Custom SVG tab butons:

http://localhost:6008/?path=/story/ag-react-tabs--custom-svg-tab-buttons
